x-pack/plugins/ml/public/explorer/explorer_charts/explorer_chart_distribution.js

```diff
@@ -38,6 +38,7 @@ import { mlChartTooltipService } from '../../components/chart_tooltip/chart_tool
 import { CHART_TYPE } from '../explorer_constants';
 
 const CONTENT_WRAPPER_HEIGHT = 215;
+const Y_AXIS_LABEL_THRESHOLD = 10;
```

Comment:
I think this needs a short comment as to what exactly it refers to

Reply:
Done in 35f8466.